src/list-issues.ts

```diff
@@ -13,10 +13,9 @@
 // limitations under the License.
 
 import * as meow from 'meow';
-import {meowFlags} from './cli';
 
-import {GitHubRepository, Issue} from './lib/github';
-import {processIssues} from './lib/asyncItemIterator';
+import {GitHubRepository, Issue} from './lib/github.js';
+import {processIssues} from './lib/asyncItemIterator.js';
```

Comment:
I'm not very knowledgeable in ESM/CJS stuff, but it seems weird to need the `.js`?

Reply:
Weird indeed, but it cannot resolve imports without it. https://nodejs.org/api/esm.html#esm_mandatory_file_extensions

TypeScript understands these imports during compilation and preserves them, so they turn out just fine in the `build` folder.